opal/views.py

```diff
@@ -71,6 +71,14 @@ def get_context_data(self, **kwargs):
         context['list_slug'] = list_slug
         context['patient_list'] = self.patient_list
         context['lists'] = PatientList.for_user(self.request.user)
+
+        context['list_group'] = None
+        if self.patient_list:
```

Comment:
If you hit say http://localhost:8000/templates/patient_list.html/opat-opat_referrals_or_something_made_up 

We check for the existence of `self.patient_list` in a bunch of places in the View - there's a good case for a PR that changes `dispatch` to raise a 404 instead of explicitly setting `None` but that's a whole different PR :)